### PR TITLE
2024.1.0

### PR DESCRIPTION
**Pull request by frenck**
## ⚠️ This PR is for reviewing purposes

### CODE REVIEW

File: source/_posts/2024-01-03-release-20241.markdown [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+the `light` to the `switch` domain.
+
+This means the entity ID also changed, and automations and dashboards may need
+to be adjusted to use the new entity ID. 
+
+If you want to go back to the light domain, use the
+[switch_as_x "Show as" option](/integrations/switch_as_x/#change-device-type-of-a-switch-from-the-existing-entity)
+on the new `Infrared lights in night mode` switch entity and select light.
+
+([@starkillerOG] - [#104713]) ([documentation](/integrations/reolink))
```
Comment (TheJulianJES) [inline source/_posts/2024-01-03-release-20241.markdown]: [#104713](https://github.com/home-assistant/core/pull/104713) seems to be the wrong PR for this change. Should be [#105916](https://github.com/home-assistant/core/pull/105916) if I'm not mistaken.

File: _config.yml [hunk 2]
```diff
@@ -108,10 +108,10 @@ social:
...
 
 # Home Assistant release details
-current_major_version: 2023
-current_minor_version: 12
-current_patch_version: 4
-date_released: 2023-12-27
+current_major_version: 2024
+current_minor_version: 1
+current_patch_version: 0
+date_released: 2023-01-03
```
Comment (gjohansson-ST) [inline _config.yml]:
```suggestion
date_released: 2023-01-03
```
```suggestion
date_released: 2024-01-03
```

Comment (frenck) [reply _config.yml]: Nice!

File: source/_posts/2024-01-03-release-20241.markdown [hunk 3]
```diff
@@ -0,0 +1,577 @@
...
+
+If you add an element, we will first show you the most used
+triggers/conditions/actions by our community. We now have all others neatly
+grouped. For actions, we took it a step further and unified them with
+service calls.
+
+### Unified actions
+
+If you have been using Home Assistant for a while, you are probably familiar
+with the “service call”-an action that you can add to your automation. But,
```
Comment (MartinHjelmare) [inline source/_posts/2024-01-03-release-20241.markdown]:
```suggestion
with the “service call”-action that you can add to your automation. But,
```

File: source/_posts/2024-01-03-release-20241.markdown [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+
+The to-do list integration keeps getting better and better. This release adds
+support for descriptions and due dates!
+
+If the integration you are using supports it, you can now add descriptions to
+each task on your list. It fully supports the Markdown markup to add links,
+bold text, etc. If due dates (or due date + time) are supported, you can
+set these.
+
+<img class="no-shadow" src='/images/blog/2024-01/todo-dialog.png' alt='Screenshot showing the new to-do item dialog, where you now can set a description and an URL.'>
```
Comment (MartinHjelmare) [inline source/_posts/2024-01-03-release-20241.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2024-01/todo-dialog.png' alt='Screenshot showing the new to-do item dialog, where you now can set a description and a URL.'>
```

File: source/_posts/2024-01-03-release-20241.markdown (hunk 1 above)
Comment (starkillerOG) [reply source/_posts/2024-01-03-release-20241.markdown]: Correct

File: source/_posts/2024-01-03-release-20241.markdown [hunk 5]
```diff
@@ -0,0 +1,577 @@
...
+It is much cleaner, all big text boxes are gone, and each item is nicely
+formatted, including an excerpt of the description and the due date. If the due
+date is overdue, it will be highlighted.
+
+## New entity type: Valve
+
+A brand new entity type has been added to Home Assistant: Valve!
+
+Integrations can use this entity type to expose valves, such as a valve on
+a radiator, pool, sprinkler systems, and main house gas or water lines.
```
Comment (MartinHjelmare) [inline source/_posts/2024-01-03-release-20241.markdown]:
```suggestion
a radiator, pool, sprinkler system, and main house gas or water line.
```

File: source/_posts/2024-01-03-release-20241.markdown [hunk 6]
```diff
@@ -0,0 +1,577 @@
...
+
+<img class="no-shadow" src='/images/blog/2024-01/valve-entity-type.png' alt='Screenshot showing an the value entity in the Home Assistant UI.'>
```
Comment (MartinHjelmare) [inline source/_posts/2024-01-03-release-20241.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2024-01/valve-entity-type.png' alt='Screenshot showing a valve entity in the Home Assistant UI.'>
```

File: source/_posts/2024-01-03-release-20241.markdown [hunk 7]
```diff
@@ -0,0 +1,577 @@
...
+current temperature in the card.
+
+<img class="no-shadow" src='/images/blog/2024-01/climate-card-change-primary-temperature.png' alt='Screenshot showing two thermostat cards. One has the setpoint as a primary temperature, the other the current temperature.'>
+
+## New card features
+
+Last year, we saw many new features added to the cards, so let's start
+the new year with some more!
+
+[@Quentame] added the climate fan mode card feature feature. This feature can be
```
Comment (MartinHjelmare) [inline source/_posts/2024-01-03-release-20241.markdown]:
```suggestion
[@Quentame] added the climate fan mode card feature. This feature can be
```

File: source/_posts/2024-01-03-release-20241.markdown [hunk 8]
```diff
@@ -0,0 +1,577 @@
...
+used with both the thermostat and the tile card:
+
+<img class="no-shadow" src='/images/blog/2024-01/climate-fan-card-feature.png' alt='Screenshot showing the new fan mode card feature the can be used on the tile and thermostat cards.'>
```
Comment (MartinHjelmare) [inline source/_posts/2024-01-03-release-20241.markdown]:
```suggestion
<img class="no-shadow" src='/images/blog/2024-01/climate-fan-card-feature.png' alt='Screenshot showing the new fan mode card feature that can be used on the tile and thermostat cards.'>
```

**Pull request merged**